packages/okta-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okta/okta-react",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
I know 3.1.2 is required here, but I thought we were bumping to latest auth-js (3.2.0/3.2.1)

Reply:
I was thinking to roll this fix out first, then update the auth-js latest with https://github.com/okta/okta-auth-js/pull/436 included.
@aarongranick-okta do you have any pending fixes to roll out for `auth-js`?

Reply:
As we discussed, removing `onSessionExpired` is a breaking change. I also believe we should change the implementation of `isAuthenticated` to assert `accessToken && idToken` to reflect realities of 3rd party cookies and other cases where the app is running - unexpectedly - with only one token. When a token renew fails it will remove the token from storage. React will be listening the event and call updateAuthState. Here we should return false from isAuthenticated if one of the tokens is missing. This will allow a new login flow to occur naturally.